README.md

```diff
@@ -1,2 +1,29 @@
-# Hello-Microverse
-my first project on microverse
+Project Name:Hello Microverse
+Description the project
+This is my first project on microverse
+Built With
+Major languages:HTML and CSS
+Frameworks:none
+Technologies used
+
+Authors
+👤 Bekalu Malede
+
+GitHub: @BEKMAL
+Twitter: @Bekalutsehay
+LinkedIn: Bekalu-Malede
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Kindly use the microverse [template](https://github.com/microverseinc/readme-template) to create your readme and ensure its neat and professional

Reply:
thank you for your comment